src/components/MdxPage/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '@tdev-hooks/useStore';
+import { useDocumentRoot } from '@tdev-hooks/useDocumentRoot';
+import { DummyMeta } from '@tdev-models/DocumentRoot';
+
+interface Props {
+    pageId: string;
+}
+
+/**
+ * This component is used to load the current page and its content.
+ */
+const MdxPage = observer((props: Props) => {
+    const pageStore = useStore('pageStore');
+    const userStore = useStore('userStore');
+    const { pageId } = props;
+    useDocumentRoot(pageId, new DummyMeta(), false);
+    React.useEffect(() => {
+        if (pageId) {
+            pageStore.addIfNotPresent(pageId, true);
+        }
+        return () => {
+            pageStore.setCurrentPageId(undefined);
+        };
+    }, [pageId]);
+    React.useEffect(() => {
+        const { current } = pageStore;
+        if (current && userStore.viewedUserId) {
+            current.loadLinkedDocumentRoots();
```

Comment:
I believe this is the call that leads to the `403` issue for non-admin users mentioned in #17. As far as I can tell, this leads to a call to `/documents` (API: `router.ts#88`), which is restricted to admins (API: `documentRoots.ts#62-64`). If I disable this line, the error goes away.

Reply:
Yes exactly: https://github.com/GBSL-Informatik/teaching-dev/pull/17#issuecomment-2401666399 (i was a bit too late ;))